### PR TITLE
Improve Test Reliability, and update tests to reflect text and structures in the current code/release

### PR DESCRIPTION
**Pull request by ctsims**

### CODE REVIEW

File: features/form_settings_tests.feature [hunk 1]
```diff
@@ -80,12 +80,12 @@ Scenario: Test the 'Change Language' setting
 	Then I select form "Question Types"
 
 	Then I select "Change Language" menu item
-	Then I see a list that contains all of these items "en,hin"
-	Then I touch the "hin" text
+	Then I see a list that contains all of these items "English,हिंदी"
```
Comment (ctsims) [inline features/form_settings_tests.feature]: @wpride I think these tests should be causing Master to fail right now, any idea why they aren't?

Comment (wpride) [inline features/form_settings_tests.feature]: I've also made these changes here https://github.com/dimagi/commcare-ui-tests/pull/57/files which is the branch targeted by AWS

Comment (wpride) [inline features/form_settings_tests.feature]: You were able to get calabash to play nicely with unicode? I had issues with the Hindi characters

Comment (ctsims) [inline features/form_settings_tests.feature]: Yup, worked fine for me on my machine. 

I did have to make sure that the file was encoded as UTF-8 without a BOM. The first time I added the unicode it defaulted to UTF-8 With BOM and that didn't work until I fixed it.

**Pull request merged**